src/main.js

```diff
@@ -104,8 +103,15 @@ function saveUserInput() {
 };
 
 function saveCurrentCover() {
-  if (!savedCovers.includes(currentCover))
+  if (!savedCovers.includes(currentCover)) {
   savedCovers.push(currentCover);
+  savedCoverSection.innerHTML +=
+  `<div class="mini-cover">
```

Comment:
This div gives the rest of the elements the proper dimensions, which are listed in the CSS on line 111 under the mini-cover class.